From c31b0a8f33b95c0077cd5ee2102a71a49bee8abe Mon Sep 17 00:00:00 2001
From: Adrian Vovk <adrianvovk@gmail.com>
Date: Wed, 23 Feb 2022 13:08:06 -0500
Subject: [PATCH] build: Fix build with meson 0.61

The positional argument was always ignored by meson. It became a
warning in meson 0.60, and a fatal error in meson 0.61

Closes: https://gitlab.gnome.org/GNOME/gnome-user-share/-/issues/27
---
 data/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/data/meson.build b/data/meson.build
index bedfe11..222c17d 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -7,7 +7,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: '@BASENAME@',
-- 
GitLab

